packages/manager/apps/pci-load-balancer/src/constants.ts

```diff
@@ -36,3 +17,17 @@ export const LISTENER_POOL_PROTOCOL_COMBINATION = {
   udp: ['udp'],
   prometheus: [],
 };
+
+export const ACTIONS = {
+  REDIRECT_TO_URL: 'redirectToURL',
+  REDIRECT_TO_POOL: 'redirectToPool',
+  REDIRECT_PREFIX: 'redirectPrefix',
+  REJECT: 'reject',
+};
+
+export const ACTION_LABELS = {
+  [ACTIONS.REDIRECT_TO_URL]: 'Redirect to URL',
```

Comment:
does it need translation ?

Reply:
It's not translated in production, it's hard-coded